fix (controls): add an offset after mouse movement

zarov opened the pull request:
When moving the globe using `GlobeControls`, there was some kind of offset, the globe not staying right under the cursor. This is visible if the container of the canvas is not occupying the full page, the container has a CSS position property different of the default `static` and there are elements right before it on the page, like the geoportal case.

The solution to avoid this, is to detect the position of the container and add an offset according to this position, using the `getBoundingClientRect` method of the container.

@elias75015 could you test this fix ?

Note: as I tweaked the `GlobeControls`, I can confirm that we need to resolve issue #545, as there were some other bugs as well.

gchoqueux commented on src/Renderer/ThreeExtended/GlobeControls.js:
```diff
@@ -904,8 +904,13 @@ function GlobeControls(view, target, radius, options = {}) {
 
             event.preventDefault();
 
+            const staticPos = window.getComputedStyle(event.target.parentElement).position !== 'static';
```
could you factorize this part that appears several times?

zarov replied:
Nope, it depends on the `event`, and this variable is available in the function scope only. As `GlobeControls` is a sort of singleton (correct me if I'm wrong), we can't compute the value at the init of the class.

gchoqueux replied:
Ok, you could pass the event as a parameter.
In short, it isn't important.

gchoqueux reviewed (commented):
Nice, thanks

gchoqueux commented:
@elias75015 can you test if the bug is fixed?

zarov committed:
fix (controls): add an offset after mouse movement

When moving the globe using GlobeControls, there was some kind of
offset, the globe not staying right under the cursor. This is visible if
the container of the canvas is not occupying the full page, the
container has a CSS position property different of the default 'static'
and there are elements right before it on the page, like the geoportal
case.

The solution to avoid this, is to detect the position of the container
and add an offset according to this position, using the
getBoundingClientRect method of the container.

pjjmunier commented:
The following bugs have been corrected :
- inconsistency of the coords picked in 3D with the mouse
- appearance of an offset between the cursor and the globe by moving the globe with the mouse
